README.md

```diff
@@ -106,7 +111,9 @@ The interaction with OpenShift is via the command line tool _oc_ which is copied
 <a name="starting-minishift"></a>
 ### Starting Minishift
 
-1. Assuming you have put _minishift_  on the _PATH_ as described in [Installing Minishift](#installing-minishift) you can start Minishift via:
+1. Run the `minishift start` command. The following example shows the result of
+the command when you start Minishift on a Linux operating system with the KVM
```

Comment:
I notice that you're asking some questions regarding the text accuracy, which I cannot answer, since I didn't touch so much the existing content but rather tried to make the flow of information more clear and do a general proofread. Some information might be outdated here but it's hard for me to determine this. If you want to request updated info feel free to write it in and I'll fix it, but for example in this case I didn't decide to use Linux as the example text and I don't know what the syntax for Windows/MacOS.